date_and_time/task2.js

```diff
@@ -0,0 +1,2 @@
+const date = new Date();
```

Comment:
Also I don't see a task from https://learn.javascript.ru/datetime

Reply:
Added this task